### ISSUE TITLE
Project scope

### ISSUE THREAD

**Issue by Riari** [bitporch/laravel-forum#86]
This is a topic that came up briefly on Slack but I think it's worth putting here to gather some opinions.

I think it's crucial that we be strict about what features get included as there are plenty that are certainly relevant to forums, but not specific to them. An important distinction between this project and, say, Flarum, is that Flarum is a standalone forum product, whereas this is a package intended to bring forum capabilities to a Laravel/Lumen application. People will install Flarum and get a forum with all the bells and whistles, but IMO that shouldn't be the case here.

So there are some things I would call out of scope, which currently includes #63 and #81 (maybe #23 and #32 as well). I think they should either be totally omitted, or developed in parallel as separate packages, or at the very least disabled/easy to remove out of the box.

Thoughts?

**Comment by AndreasElia** [comment]
Having thought about it some more over the past few days I agree. I still think we should include statistics (#23), even if never visible on the frontend. As for #32 we only really need an admin interface for managing groups imo, we can figure out another way of doing this or leave it to the end user to integrate within their own application.

**Comment by Dylan-DPC** [comment]
Let's define in this issue what all we want to cover in MVP. That way we can list what is left to be done

**Comment by AndreasElia** [comment]
I feel everything else assigned to the 1.0 milestone would be great for MVP.

**Comment by Dylan-DPC** [comment]
that's true but we still don't know what is left

**Comment by AndreasElia** [comment]
Pretty sure we've got this sorted out now.
